<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aiuto</class>
 <widget class="QDialog" name="Aiuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Grafico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Icona_da_campo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Descrizione_Scudetto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="Descrizione_Civetta">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>200</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Civetta">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="Close">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>740</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Occhio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Coccinella">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Manometro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Descrizione_Occhio">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="Frase_di_Cortesia">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>891</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Frase</string>
   </property>
  </widget>
  <widget class="QLabel" name="Descrizione_Grafico">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="Descrizione_Ruota_Dentata">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Scudetto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Descrizione_Mano_ceh_saluta">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="Descrizione_Manometro">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>630</y>
     <width>591</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Ruota_dentata">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Immagine_Mano_che_saluta">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Immagine</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Descrizione_Coccinella">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>340</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="Descrizione_Icone_da_Campo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
